sites/hashdev/src/_pages/blog/10_error-stack-update-0-2.mdx

```diff
@@ -0,0 +1,185 @@
+---
+authorPhoto: blog/authors/tim-diekmann.jpg
+postPhoto: blog/10_error-stack-update-0-2/Cover.jpg
+title: "Announcing error-stack v0.2"
+subtitle: "Redesigned and customizable outputs, related errors and multiple sources, and much more"
+author: "Tim Diekmann"
+jobTitle: "Platform Engineer, HASH"
+date: "2022-10-03"
+---
+
+Back in June [we announced the first version](https://hash.dev/blog/announcing-error-stack) of `error-stack`, our context-aware error library. We developed this to help address a lot of challenges we encountered when implementing error handling in our simulation engine [LINK]. In the time since then we’ve started using it in our entity-graph query layer [LINK?] that we’re building as a datastore for HASH [LINK?]. Across that development we’ve learned a lot of lessons, and in joint efforts with open-source contributors we’re ready to release a new version of the library.
```

Comment:
Do we want to link the entity-graph query layer?

Reply:
What would we link to here?

Reply:
I think mainly to the dev/graph work, but as this is a rather temporary link target, we should probably just don't add a link, as it's basically included in `HASH` a few words later.

Reply:
True, without a better place to link to I definitely wouldn't do it now.